newIDE/app/src/SceneEditor/SwipeableDrawerEditorsDisplay/UseSwipeGesture.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from 'react';
+
+type Props = {|
+  onSwipeUp: () => void,
+  onSwipeDown: () => void,
+|};
+
+const useSwipeGesture = (props: Props) => {
+  const startTimeRef = React.useRef<?number>(null);
+  const startYRef = React.useRef<?number>(null);
+
+  const onTouchStart = React.useCallback((event: TouchEvent) => {
+    startTimeRef.current = Date.now();
+    startYRef.current = event.touches[0].clientY;
+  }, []);
+
+  const onTouchEnd = React.useCallback(
+    (event: TouchEvent) => {
+      if (!startYRef.current || !startTimeRef.current) return;
+      const { current: startY } = startYRef;
+      const { current: startTime } = startTimeRef;
+
+      const deltaY = event.changedTouches[0].clientY - startY;
+      const deltaTimeInSeconds = (Date.now() - startTime) / 1000;
+      if (
+        Math.abs(deltaY) > 30 &&
```

Comment:
maybe "variable-ize" those 2 constants minMovement & minSpeed?

Reply:
Oh yeah, forgot about that